.github/workflows/helm-test.yml

```diff
@@ -127,15 +128,15 @@ jobs:
           fi
           sudo install /tmp/minikube /usr/local/bin/minikube
           sudo chmod a+x /usr/local/bin/minikube
+          sudo sysctl fs.protected_regular=0
 
       - name: Start minikube
-        env:
-          MINIKUBE_IN_STYLE: "false"
-          MINIKUBE_WANTUPDATENOTIFICATION: "false"
-          MINIKUBE_WANTREPORTERRORPROMPT: "false"
-          CHANGE_MINIKUBE_NONE_USER: "true"
-        run:
-          sudo "PATH=$PATH" -E /usr/local/bin/minikube start --vm-driver=none --wait=true --extra-config=kubeadm.ignore-preflight-errors=FileExisting-crictl --kubernetes-version=${{ matrix.K8S_VERSION }}
+        run: |
+          minikube start --container-runtime=docker --cpus 2 --memory 4096 --kubernetes-version=${{ matrix.K8S_VERSION }} --no-vtx-check
```

Comment:
Random related nit: Minikube that sets up Kubernetes 1.25.0 has a known issue with resolv.conf that was fixed in Kubernetes 1.25.1. I got hit with this issue. Might be worth upgrading K8S_VERSION to version +1.25.1 or verifying this issue is fixed in your version of Minikube.

https://github.com/kubernetes/kubernetes/issues/112135